lib/auth/vault.js

```diff
@@ -270,7 +265,7 @@ function checkHealth(log, callback) {
         defResp[implName] = { code: 200, message: 'OK' };
         return callback(null, defResp);
     }
-    return client.healthcheck(log, (err, obj) => {
+    return client.healthcheck(log.getSerializedUids(), (err, obj) => {
```

Comment:
This change is needed for https://github.com/scality/vaultclient/pull/136 since the werelogs method `unserializeUids` requires a request UID, and not a logger instance.